transport_nantes/photo/views.py

```diff
@@ -85,3 +91,257 @@ def get_context_data(self, **kwargs):
                 last_submitted_photo.submitted_photo.url
 
         return context
+
+
+class PhotoView(FormView):
+    """
+    View to display a single photo and up/down vote it
+    """
+    template_name = 'photo/single_entry.html'
+    form_class = AnonymousVoteForm
+
+    def get(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
+        """Overriding here so we can return HttpResponseForbidden if needed"""
```

Comment:
This is assuredly the wrong way to do this.  If there's no way to throw a 403 from `get_context_data()` (where you determine that this is the right action) and have django do the right thing, then define a 403 exception yourself and catch it here.  Don't override return types, it's twisted and makes your code harder to understand.

Reply:
Fixed that with a custom error